team-contract.md

```diff
@@ -0,0 +1,45 @@
+# Team Contract (Project-Avalon)
+
+_Date: 2023-11-10_
+
+## Expectations
+
+The time expected for group members to work on DSCI 522 will be 8-10 hours per week/per person, unless all the team members agree to lengthen the time expected. If team members cannot complete their sections within the maximum hours, members should give notice on Slack at least 24 hours before the group submission date (Saturday 6am) to see if work should be redistributed or if the expected time should be increased.
+
+All submissions are expected to meet the criteria of the milestones as set out in the instructions. This includes making sure all scripts are working, creating tests for all your functions, and using proper grammar in written submissions.
+
+## Meetings
+
+Meeting times were established during the initial team meeting held on 2023-10-09. Meetings will be held on Friday.
+
+The group will have a second formal meeting during the lab for 522 on Monday at 2 pm. All members are expected to attend the meetings.
+
+## Communication
+
+All communication outside of the meetings will be through Slack or Github. Github will be the primary communication tool for ideas, suggestions, submitting work, and reviewing work. Slack will be the primary communication tool for informal communication and scheduling meetings.
+
+## Reviews
+
+All members are expected to submit a pull request for their work on the repository for the group project on Github.com. Members are expected to add teammates as reviewers to have all pull request reviewed by another teammate. Pull requests should be reviewed within 24 hours after creation.
+
+## Roles
+
+There are two roles team members will be responsible for each week. The roles are as follows:
+1. **Scribe**:
+   - Responsible for taking notes at the meetings and in-class lab sessions and adding as a GitHub issue.
+   - Responsible for sharing all notes and in-class activities with teammates.
+2. **Proof-reader**:
+   - Responsible for doing a final check of the submission before the deadline to make sure all requirements are satisfied.
+
+Positions will be rotated each week to ensure that each team member does each position once.
```

Comment:
Will we assume these roles? I'm sorry but I don't remember we discussed this on the lab...

Reply:
We did not discuss about it, but we need someone to double check it before submitting

Reply:
I just converted the pdf to .md. let me ask who created the pdf